### PR TITLE
added file joh-sto-hello-world

### PR DESCRIPTION
**Pull request by johsto**

### CODE REVIEW

**Review by mohammed-softordi — CHANGES_REQUESTED** [review]
Need some changes

File: joh-sto-hello-world.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+echo "hello world"
```
Comment (mohammed-softordi) [inline joh-sto-hello-world.md]: Change something

**Review by mohammed-softordi — CHANGES_REQUESTED** [review]
Do some changes

**Review by mohammed-softordi — APPROVED** [review]

**Comment by johsto** [comment]
merge after changes

**Pull request merged**